src/index.ts

```diff
@@ -2,3 +2,7 @@ import { Factory } from './factory';
 
 export default new Factory().newBuilder();
 export type { JUnitReportBuilder as Builder } from './builder';
+export type { TestSuite } from './test_suite';
+export type { TestCase } from './test_case';
+export type { TestGroup } from './test_group';
```

Comment:
If I recall correctly, `TestGroup` could/should be `abstract`, and if so it should not be exported here. But I'll confirm and change that myself.